pyproject.toml

```diff
@@ -4,7 +4,7 @@ requires = ["setuptools",
             "wheel",
             "extension-helpers",
             "oldest-supported-numpy",
-            "cython==0.29.14"]
+            "cython"]
```

Comment:
should pin it to a version so that the generated C code is guaranteed stable. You can increase it to any released version if you need to though.

Reply:
Fixed.